lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -129,10 +150,22 @@ def load
         records = ActiveRecord::Bitemporal.with_bitemporal_option(**bitemporal_option) { super }
 
         return records if records.empty?
-        return records unless ActiveRecord::Bitemporal.valid_datetime || bitemporal_value[:with_valid_datetime] && bitemporal_value[:with_valid_datetime] != :default_scope && valid_datetime_
+
+        valid_datetime_ = valid_datetime
```

Comment:
Is this duplicate?
https://github.com/kufu/activerecord-bitemporal/blob/19d5a021b4a83fa52a478fe7c3eb00dab99a16a7/lib/activerecord-bitemporal/bitemporal.rb#L147

Reply:
Thanks! :)
I removed it.
https://github.com/kufu/activerecord-bitemporal/pull/82/commits/5181ec36bb0839fc60ec3075e4e7348527b6f5e7